src/automerge-repo-undo-redo.ts

```diff
@@ -174,7 +174,7 @@ export class AutomergeRepoUndoRedo<T> {
             heads,
           );
 
-          change.redo.patches = unpatchAll(before, patches);
+          // change.redo.patches = unpatchAll(before, patches);
```

Comment:
this one fixes the `can apply redos and undos` but at the same time it breaks the `local undo + global redo` test